internal/datadog/go.mod

```diff
@@ -0,0 +1,72 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/internal/datadog
+
+go 1.19
+
+require (
+	github.com/DataDog/datadog-agent/pkg/trace v0.46.0
```

Comment:
This should match the version used here https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/19f2a53e42f9ace56e14a624fdaa56a482082cb8/exporter/datadogexporter/go.mod#L7 can you bump the version?

Reply:
This still needs addressing

Reply:
I believe this is causing the CI failures.